enterprise/management/commands/send_groups_reminder_email.py

```diff
@@ -0,0 +1,59 @@
+"""
+Django management command for sending reminder emails to pending users to accept groups invitation.
+"""
+
+import logging
+
+from django.contrib.auth import get_user_model
+from django.core.management import BaseCommand
+from enterprise.models import EnterpriseGroupMembership
+from enterprise.tasks import send_group_reminder_emails
+
+User = get_user_model()
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Django management command for sending reminder emails to learners that have been added to a group
+
+    This command sends reminder emails to learners at the 5, 25, 50, 65, and 85 day mark before the 90
+    day purge date.
+
+    Example usage:
+    """
+
+    def add_arguments(self, parser):
+        """
+        Entry point to add arguments.
+        """
+        parser.add_argument(
+            '--dry-run',
+            action='store_true',
+            dest='dry_run',
+            default=False,
+            help='Dry Run, print log messages without spawning the celery tasks.',
+        )
+
+        parser.add_argument(
+            '--days_before_purge_date',
+            type=int,
+            dest='days_before_purge_date',
+            default=30,
+            metavar='NUM_DAYS',
+            help='The amount of days before the 90 day purge pii date to send a braze reminder email for pending users to accept group memebership',
+        )
+
+    def handle(self, *args, **options):
+        """
+        Command's entry point.
+        """
+        dry_run = options['dry_run']
+        days_before_purge_date = options['days_before_purge_date']
+
+        # get all group memberships
+        for group_membership in EnterpriseGroupMembership.objects.all():
+            # loop through pending customer list in each group membership
+            for pending_enterprise_customer_user in group_membership.pending_enterprise_customer_users:
```

Comment:
a group membership doesn't have a list of pending users, it either has a single pending user or a realized user. You can actually do a filter for only memberships that apply- something like
```
EnterpriseGroupMembership.objects.filter(enterprise_customer_user__isnull=True)
```
That way you're only grabbing memberships that have not been realized. The full line would look something like 
```
for pending_user in EnterpriseGroupMembership.objects.filter(enterprise_customer_user__isnull=True).values_list('pending_enterprise_customer_user', flat=True):
```

Reply:
FWIW I _**also**_ think you could do the filtering for your created_at times here as well! You can create a query `Q` object that ors the operations of checking the `created_at` value on the table.

Reply:
LMK if you wanna chat about it!